ui/package.json

```diff
@@ -33,7 +33,7 @@
     "zone.js": "^0.11.8"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^14.2.1",
+    "@angular-devkit/build-angular": "^14.2.10",
```

Comment:
Angular devkit 14.2.1 required `loader-utils@3.2.0`, this version updated to `loader-utils@3.2.1`.